lib/usb-hid-msc-lib.robot

```diff
@@ -33,3 +33,23 @@ Download ISO And Mount As USB
             Skip    unsupported
         END
     END
+
+Mount ISO As USB
+    [Documentation]    Mounts an ISO file from the local sources. In case of
+    ...    PiKVM we need to manually upload images to the device. For UEFI
+    ...    Secure Boot tests images, this can be automatically done by running
+    ...    `scripts/secure-boot/generate-images/sb-img-wrapper.sh` script.
+    [Arguments]    ${img_path}
+
+    ${img_dir}    ${img_name}=    Split Path    ${img_path}
+
+    IF    "${MANUFACTURER}" == "QEMU"
+        Remove Drive From Qemu
+        Add USB To Qemu    img_name=${img_path}
+    ELSE
+        IF    "${DUT_CONNECTION_METHOD}" == "pikvm"
+            Mount Image On PiKVM    ${PIKVM_IP}    ${img_name}
+        ELSE
+            Skip    unsupported
+        END
```

Comment:
Any informative error message on other platform? Or we already ensure this would not be called in such a case?

Reply:
Tried to add such msg https://github.com/Dasharo/open-source-firmware-validation/pull/187/commits/13e2018fef5b3b1f047a06eb4538705a8f4c1141 but to be honest, I think that whole IF with pikvm should be gone, because for real hardware we always use it